bps/bps_ApPipe.yaml

```diff
@@ -10,11 +10,25 @@ campaign: smallrun_example
 
 submitPath: ${PWD}/bps/{outCollection}
 computeSite: ncsapool
+# The default BPS walltime is on the order of 3 days.
+# That's much more than the vast majority of jobs need,
+# and it can cause your jobs not to run if you're too close to a maintenance window.
+# Set the default walltime as appropriate for what you are running (12 hours below).
+site:
+  ncsapool:
+    profile:
+      condor:
+        +Walltime: 43200
 # Memory allocated for each quantum, in MBs
 requestMemory: 2048
 # CPUs to use per quantum
 requestCpus: 1
```

Comment:
The likelihood that requestCpus will be changed any time soon is very small. And this is the default value. Could leave out of this submission. requestMemory is also the same as the current default, but if want to keep it here as a reminder it shouldn't cause a problem (unless the majority of the Quanta suddenly start taking more than that)

Reply:
Yes, I think both requestMemory and requestCpus were both in there as reminders.

Reply:
I'd prefer you leave out `requestCpus` and leave `requestMemory`, with a note that this is the bps default.